common/djangoapps/third_party_auth/tests/test_views.py

```diff
@@ -153,3 +155,42 @@ def test_login_disabled(self):
         self.enable_saml(enabled=False)
         response = self.client.get(self.LOGIN_URL)
         self.assertEqual(response.status_code, 404)
+
+
+@unittest.skipUnless(AUTH_FEATURE_ENABLED, AUTH_FEATURES_KEY + ' not enabled')
```

Comment:
This is necessary to prevent these tests from running with the CMS environment file. The [LMS test environment file](https://github.com/edx/edx-platform/blob/master/lms/envs/test.py#L232) has a handful of authentication settings that the CMS test environment file doesn't, so these tests fail when run with the CMS environment file, which causes the Python unit tests to fail on Jenkins.